src/action/common/mod.rs

```diff
@@ -1,25 +1,29 @@
 //! [`Action`](crate::action::Action)s which only call other base plugins
 
-#[cfg(all(target_os = "macos", not(feature = "nix-community")))]
-pub(crate) mod configure_enterprise_edition_init_service;
+#[cfg(not(feature = "nix-community"))]
+pub(crate) mod configure_determinate_nixd_init_service;
 pub(crate) mod configure_init_service;
 pub(crate) mod configure_nix;
 pub(crate) mod configure_shell_profile;
+pub(crate) mod configure_upstream_init_service;
 pub(crate) mod create_nix_tree;
 pub(crate) mod create_users_and_groups;
 pub(crate) mod delete_users;
 pub(crate) mod place_nix_configuration;
+pub(crate) mod provision_determinate_nixd;
 pub(crate) mod provision_nix;
 pub(crate) mod setup_channels;
 
-#[cfg(all(target_os = "macos", not(feature = "nix-community")))]
-pub use configure_enterprise_edition_init_service::ConfigureEnterpriseEditionInitService;
+#[cfg(not(feature = "nix-community"))]
+pub use configure_determinate_nixd_init_service::ConfigureDeterminateNixdInitService;
 pub use configure_init_service::{ConfigureInitService, ConfigureNixDaemonServiceError};
 pub use configure_nix::ConfigureNix;
 pub use configure_shell_profile::ConfigureShellProfile;
+pub use configure_upstream_init_service::ConfigureUpstreamInitService;
 pub use create_nix_tree::CreateNixTree;
 pub use create_users_and_groups::CreateUsersAndGroups;
 pub use delete_users::DeleteUsersInGroup;
 pub use place_nix_configuration::PlaceNixConfiguration;
+pub use provision_determinate_nixd::ProvisionDeterminateNixd;
```

Comment:
```suggestion
#[cfg(not(feature = "nix-community"))]
pub use provision_determinate_nixd::ProvisionDeterminateNixd;
```

Reply:
I don't think we tried to exclude these before, but doing so will cause comp errors in additional files that we have to go clean up. I think we still want to just lean on disabling the CLI option that leads here for these as well?